Glyphs/FunctionalComponents/complex/README.md

```diff
@@ -6,9 +6,9 @@ SBO:0000253 Non-covalent complex
 ## Recommended Glyph and Alternates
 The RECOMMENDED glyph for a complex is a composite of the glyphs for the molecules comprising the complex.  For example, a protein bound to a simple chemical, a guide RNA, or another protein (in order:protein-small molecule, protein-guide RNA, protein-protein):
 
-![glyph example](complex-ps-specification.png)
-![glyph example](complex-pr-specification.png)
-![glyph example](complex-pp-specification.png)
+![glyph specification](complex-ps-specification.png)
+![glyph specification](complex-pr-specification.png)
+![glyph specification](complex-pp-specification.png)
```

Comment:
Should these really be taken as specifications? They are particular examples of compositions, which could be used canonically, but they are only a few of the many potential compositions.